src/main/types/pageURL.ts

```diff
@@ -2,6 +2,8 @@ const SEPARATOR = "/";
 
 const SOLE_TRADER = "/sole-trader";
 
+export const BASE_URL = "/register-acsp";
```

Comment:
Instead of concatenating  URLs in controller can concat them in pageURL.ts please? that will keep changes i one file and easier to maintain

Reply:
Yes I have asked Benedict to create a tech debt ticket to move urls in PageURLs file. As it needed a lot of refactoring and I didn't want to cloud this PR with that work.